english/call-a-doctor/doctor.md

```diff
@@ -3,7 +3,7 @@ As you exhale, the walls stiffen, becoming less like marshmallow, and more like
 the interior of your cell. The walls are a toasted-marshmallow kind of dirty white 
 color, not like the institutionalized white you remember from movies.
 
-The doctor enters.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.